src/config.ts

```diff
@@ -78,18 +86,22 @@ export const CONFIG = {
         },
       },
     },
-    title: {
+    descriptionValidation: {
       fullPrompt: {
         row: 16,
         col: 2,
       },
-      preferGeneratedValues: {
-        row: 19,
-        col: 2,
+      maxRetries: {
+        row: 14,
+        col: 8,
       },
-      allowedWords: {
-        row: 20,
-        col: 2,
+      minScore: {
+        row: 14,
+        col: 7,
+      },
+      keyword: {
```

Comment:
see above, I would be in favor of removing this altogether

Reply:
if "Score:" was missing from prompt, it was virtually impossible to parse the output of evaluation, PLUS, parsing in Polish required to change the noun to something more local. and the parsing code needs to be aware if the word. I'd leave it in.